### PR TITLE
ATR-305: do not dig into methods of PXGraph, PXView, and PXCache; new…

### PR DESCRIPTION
**Pull request by metallmonster**
… bypassMethod parameter in the constructor of NestedInvocationWalker

### CODE REVIEW

File: src/Acuminator/Acuminator.Utilities/Roslyn/NestedInvocationWalker.cs [hunk 1]
```diff
@@ -58,14 +59,40 @@ public abstract class NestedInvocationWalker : CSharpSyntaxWalker
...
             CancellationToken = cancellationToken;
 
+			if (bypassMethod != null)
+			{
+				_bypassMethod = bypassMethod;
+			}
+			else
+			{
+				var pxContext = new PXContext(_compilation);
+				var typesToBypass = new[]
```
Comment (SENya1990) [inline src/Acuminator/Acuminator.Utilities/Roslyn/NestedInvocationWalker.cs]: @metallmonster maybe it makes sense to refactor out creation of forbidden types register - put it into separate field or generate in a virtual method. This way the inheritors of nested walker will have the ability to extend the set of types to which they should not descend

Comment (metallmonster) [reply src/Acuminator/Acuminator.Utilities/Roslyn/NestedInvocationWalker.cs]: Ok

**Comment by metallmonster** [comment]
@SENya1990 one more try

**Comment by metallmonster** [comment]
@dropsonic please merge if everything is ok

**Pull request merged**